src/assets/scripts/client/base/Compass.js

```diff
@@ -0,0 +1,57 @@
+import { radians_normalize } from '../math/circle';
+
+/**
+ * @class Compass
+ */
+class Compass {
+    /**
+     * @constructor
+     */
+    // istanbul ignore next
+    constructor() {
```

Comment:
Why `// istanbul ignore next`?

Reply:
copy-n-paste ... I guess from the AirportModel

Reply:
it's because the version of nyc (which wraps istanbul) used in the project has a hard time with `constructors`.  meaning, sometimes the constructor all on it's own would show as uncovered in coverage reports.  this comment tells istanbul to simply ignore the line.

probably fixed in more recent versions, since es2015+ is much more of a standard than it was 2-3 years ago when we moved to es2015.  see #1137

Reply:
so...do you want me remove the line or should we keep it for now?